requirements.txt

```diff
@@ -4,3 +4,4 @@ django-countries
 stripe
 crispy-bootstrap5
 python-decouple
+pillow
```

Comment:
I think pillow is not used in the project

Reply:
The pillow dependency is necessary to check the size of the images